docs/support/tor.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Tor
+parent: Supporting Applications
+nav_order: 720
+---
+## Tor: Private Networking
+
+Running the BitBox Base means connecting to other Bitcoin and Lightning Network peers.
+If done over the regular internet protocol (IPv4), your IP address is visible to all your communication partners.
+As IP addresses can be located geographically, this means revealing your physical location, from the general area down to individual city blocks (the granularity depends on how your internet service provider manages IP addresses).
+You can easily locate yourself using services like <https://iplocation.io> or <http://www.infosniper.net>.
+
+Revealing your location with the information that you are using Bitcoin is not a good idea. This is why the BitBox Base uses [The Onion Router](https://www.torproject.org/) (Tor) to communicate to other Bitcoin peers privately and without revealing your real IP address.
+
+By creating "hidden services", Tor can also be used to securely access your BitBox Base from outside your local network without having to configure any networking devices, such as port-forwarding on your router. 
+
+Please not that only Bitcoin-related communication is routed over Tor.
+
+### Installation
+
+Tor is installed using the official repository at `https://deb.torproject.org/torproject.org`, over a TLS connection secured by the official Tor signing key.
+
+### Configuration
+
+The Tor configuration is application-specific:
+
+* **Bitcoin Core**
+  * Currently, `bitcoind` is not configured to listen to other Tor nodes.
+  * To connect to other Bitcoin nodes, all communication is routed over the Tor SOCKS proxy. This configuration is set in `/etc/bitcoin/bitcoin.conf` with the option `proxy=127.0.0.1:9050`. Peers will see a regular IP address, but one that belongs to the Tor network and not your own.
+  * For initial peer discovery on the Tor network, well-known `.onion` nodes are provided to get additional addresses from:  
```

Comment:
Do we have any info on where we found this list of "well-known" `.onion` nodes?

Reply:
Unfortunately not, these are used everywhere in scripts on the internet, maybe @jonasschnelli knows them?

Reply:
I'll open an issue for that, but I think it's safe to merge this as-is.

Reply:
https://github.com/shiftdevices/bitbox-base-internal/issues/180

Reply:
Okay, sounds good. Resolving this thread.